website/intro/use-cases.mdx

```diff
@@ -1,100 +1,92 @@
 
-You can use Terraform to codify the configuration for software defined networks.
-Terraform can then use this configuration to automatically set up and modify settings by interfacing with the control layer. This allows the configuration to be
-versioned and changes to be automated. For example, you can [use Terraform to configure AWS VPC](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/vpc).
+### Resources
 
-## Resource Schedulers
+- Try the [Network Infrastructure Automation with Consul-Terraform-Sync Intro](https://learn.hashicorp.com/tutorials/consul/consul-terraform-sync-intro?in=consul/network-infrastructure-automation) tutorial on Hashicorp Learn to install Consul-Terraform-Sync on a node. You will then configure it to communicate with a Consul datacenter, react to service changes, and execute an example task.
+- Try the [Consul-Terraform-Sync and Terraform Enterprise/Cloud Integration](https://learn.hashicorp.com/tutorials/consul/consul-terraform-sync-terraform-enterprise?in=consul/network-infrastructure-automation) tutorial on HashiCorp Learn to configure Consul-Terraform-Sync to interact with Terraform Enterprise and Terraform Cloud.
 
-In large-scale infrastructures, static assignment of applications to machines
-becomes increasingly challenging. To solve that problem, there are a number
-of schedulers like Borg, Mesos, YARN, and Kubernetes. These can be used to
-dynamically schedule Docker containers, Hadoop, Spark, and many other software
-tools.
 
-Terraform is not limited to physical providers like AWS. Resource schedulers
-can be treated as a provider, enabling Terraform to request resources from them.
-This allows Terraform to be used in layers: to setup the physical infrastructure
-running the schedulers as well as provisioning onto the scheduled grid.
+## Kubernetes
+
+Kubernetes is an open-source workload scheduler for containerized applications. Terraform lets you both deploy a Kubernetes cluster and manage its resources (e.g., pods, deployments, services, etc.). You can also use the [Kubernetes Operator for Terraform](https://github.com/hashicorp/terraform-k8s) to manage cloud and on-prem infrastructure through a Kubernetes custom resource definition (CRD) and Terraform Cloud.
+
+### Resources
+
+- Try the [Manage Kubernetes Resources via Terraform](https://learn.hashicorp.com/tutorials/terraform/kubernetes-provider?in=terraform/kubernetes) tutorial on HashiCorp Learn. You will use Terraform to schedule and expose a NGINX deployment on a Kubernetes cluster.
+- Try the [Deploy Infrastructure with the Terraform Cloud Operator for Kubernetes](https://learn.hashicorp.com/tutorials/terraform/kubernetes-operator) tutorial on HashiCorp Learn. You will configure and deploy the Operator to a Kubernetes cluster and use it to create a Terraform Cloud workspace and provision a message queue for an example application.
+
```

Comment:
We also have this tutorial for CRDS: https://learn.hashicorp.com/tutorials/terraform/kubernetes-crd-faas

Reply:
oh thank you! I think I will keep it to 2 here just so I don't overwhelm this page, but I do appreciate the suggestion!